node/graphql/index.ts

```diff
@@ -38,11 +38,11 @@ class BookDatabase {
     return false
   }
 
-  public add({authors, name}) {
+  public add({authors, name, id}) {
```

Comment:
You don't need `id` here. Ids should not be an argument for object creation, but rather be auto-generated by the database